src/lint/lint.ts

```diff
@@ -36,13 +37,12 @@ export async function validateCustom(
     // fs.readFile), so the easiest way seems to be to convert the AMF error to
     // a string and then work with that string.
 
-    const message = `${err.message ?? err}`;
-    if (message.includes("no such file or directory")) {
```

Comment:
This code was removed because it was swallowing errors related to a missing library instead of just a missing profile.